gammapy/data/tests/test_data_store.py

```diff
@@ -211,3 +211,10 @@ def test_make_mean_edisp(tmpdir):
     i = np.where(rmf.data.evaluate(e_reco=Energy(40, "TeV")) != 0)[0]
     i2 = np.where(rmf2.data.evaluate(e_reco=Energy(40, "TeV")) != 0)[0]
     assert_equal(i, i2)
+
+
+@requires_data('gammapy-extra')
```

Comment:
You have to add `@requires_dependency('yaml')` here: https://travis-ci.org/gammapy/gammapy/jobs/249970758#L1794